docs/how-to/uc-image-creation.rst

```diff
@@ -0,0 +1,309 @@
+How to create a Real-time Ubuntu Core image
+===========================================
+
+.. note::
+
+    This guide assumes access to features available exclusively to `dedicated Snap Store`_ users.
+
+Canonical builds and publishes Ubuntu Core images for a range of `supported platforms`_.
+However, these images include the generic Linux kernel. 
```

Comment:
I would say "Ubuntu kernel" instead of "LInux kernel". We also published optimised Ubuntu Core images for supported platforms. I suggest rephrasing this to read: "Canonical builds and publishes generic and silicon-optimised Ubuntu Core images for a range of supported platforms. The generic Ubuntu Core images include the generic Ubuntu kernel by default. In order [...]"

Reply:
We've referred to the kernel as Linux kernel and realtime kernel across the documentation. Changing it in one place will add confusion. I've addressed the rest in https://github.com/canonical/real-time-ubuntu-docs/pull/45/commits/f2983be06138d9c9f8d2835d321581a29429e3b5

Reply:
Changed to [Ubuntu kernel](https://ubuntu.com/kernel): https://github.com/canonical/real-time-ubuntu-docs/pull/45/commits/9f3351bde8c653ba3d61768e007906bb0d64423e